pages/createPledge.js

```diff
@@ -391,6 +393,7 @@ class CreatePledgePage extends React.Component {
                             id="slug"
                             name="slug"
                             defaultValue={slugify(name || '').toLowerCase()}
+                            data-cy="slug"
```

Comment:
Let's be a bit more specific with them

```suggestion
                            data-cy="slugInput"
```

Reply:
made the change